.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -108,4 +108,4 @@ steps:
           slurm_time: 15:00:00
         env:
```

Comment:
Could we also extend the time for the regional run? It keeps failing due to that - maybe 45 minutes?